src/containers/Media/Media.js

```diff
@@ -25,7 +25,17 @@ export default class extends Component {
   render() {
     return (
       <div className={style.block}>
-        <img className={style.image} alt={this.props.image} src={this.props.media.selectedImage || this.props.image} />
+        <picture className={style.image}>
+          {/*<!--[if IE 9]><video style="display: none;"><![endif]-->*/}
+          <source alt={this.props.image} srcSet={this.props.media.selectedImage || this.props.image[0] + ', ' + this.props.image[1] + ' 2x'} media="(min-width: 400px)" />
+          {/*<!--[if IE 9]></video><![endif]-->*/}
+          <img
+            alt={this.props.image}
+            className={style.image}
+            srcSet={this.props.media.selectedImage || this.props.image[0] + ', ' + this.props.image[1] + ' 2x'}
+            src={this.props.media.selectedImage || this.props.image[0]}
```

Comment:
I would actually say that this is ok. At least the fallback is working. Let's punish the IE11 users